cli/internal/cmd/apply.go

```diff
@@ -227,7 +227,7 @@ func runApply(cmd *cobra.Command, _ []string) error {
 	}
 
 	fileHandler := file.NewHandler(afero.NewOsFs())
-	logger, err := newDebugFileLogger(cmd, fileHandler)
+	debugLogger, err := newDebugFileLogger(cmd, fileHandler)
```

Comment:
I don't understand why we have two loggers in this func. Should we just log everything with this logger?

Reply:
The other logger `log` is now only used in the warnLogger, which in turn is only used https://github.com/edgelesssys/constellation/blob/euler/cli/extend-file-debug-log/cli/internal/cmd/applyinit.go#L33. I think in general terms, we want to print warning non-structured, so not with slog. 


The simplest solution would be to use the debugLogger instead of log when creating the warnLogger and in the warnLogger's implementation we debug log additionally when calling Warn.

Reply:
The current state (last commit) is a proposal for this version.

Reply:
I agree that printing the warning as structured debug log again is fine.